estuary/utils/story.py

```diff
@@ -150,8 +150,9 @@ def get_wait_times(self, results):
         Get the wait time between two different artifacts.
```

Comment:
This doc string is a bit misleading. Could you clarify that this is for the whole story?